onnxruntime/contrib_ops/cuda/quantization/qordered_ops/qordered_qdq.cc

```diff
@@ -95,7 +95,7 @@ cublasLtOrder_t GetCublasLtOrderAttr(const OpKernelInfo& info, const char* order
 #endif
 
 QuantizeWithOrder::QuantizeWithOrder(const OpKernelInfo& info) : CudaKernel(info) {
-#if defined(CUDA_VERSION) && CUDA_VERSION >= 11040
+#if defined(USE_CUDA)
```

Comment:
remove #else below

Reply:
And they do not need to support ROCM?

Reply:
How about the other files in the same directory?

Reply:
qordered_ops files are excluded from ROCm so I think it is fine: https://github.com/microsoft/onnxruntime/blob/f0139658316ce231aa5db54b3bc575cd85c6dd4c/cmake/onnxruntime_rocm_hipify.cmake#L58